### ISSUE TITLE
A11Y : People List Page in grid view

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/tiamat-theme#904]
There isn't enough space to render the full email address (e.g. jeremy.sparks@colorado.edu)  of a user on the table view particularly in a mobile view and because of the way email address are structured there's no great place to allow them to break naturally.  

On the grid view we need to switch to rendered output of the email address to just read the word `email` and be a link to the correct email address rather than the full email address text.  

Additionally we want to make sure that the images do not get too small.  There should be some minimum width set on the images in table view.

**Comment by timurtripp** [comment]
Additionally: Image missing alt text, link present twice (once for the image and once for the name).
